manifests/kustomize/base/cache-deployer/cache-deployer-statefulset.yaml

```diff
@@ -20,5 +25,5 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
       serviceAccountName: kubeflow-pipelines-cache-deployer-sa
-      restartPolicy: OnFailure
+      restartPolicy: Always
```

Comment:
Can you please explain the reason to restart a successfully completed deployer?

Reply:
StatefulSet and Deployment only support restartPolicy: Always. In this scenario the script will keep running. Seems like Job is still the most fit role. But regarding upgrade issue, is there any way we can delete the deployer-job after it finish?